weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SimplePageLayout.tsx

```diff
@@ -218,30 +341,58 @@ export const ScrollableTabContent: React.FC<{
   );
 };
 
+const isPrimitive = (val: any) => {
```

Comment:
This is defined multiple times, we should probably pull it out into a utility file

Reply:
Done